tests/resources/dynamic-vars.py

```diff
@@ -31,9 +31,9 @@ def getEnvironment():
         return TestEnvironment.LOCAL
 
 def getName(image):
-    return {TestEnvironment.DRONE: 'harbor.ci.drone.local/library/{}'.format(image),
+        return {TestEnvironment.DRONE: 'wdc-harbor-ci.eng.vmware.com/default-project/{}'.format(image),
             TestEnvironment.LONGEVITY: 'vic-executor1.vcna.io/library/{}'.format(image),
-            TestEnvironment.LOCAL: image}[getEnvironment()]
+            TestEnvironment.LOCAL: 'wdc-harbor-ci.eng.vmware.com/default-project/{}'.format(image)}[getEnvironment()]
```

Comment:
I don't think we want to make this local change

Reply:
should still just point to docker hub

Reply:
Done